api/v1alpha1/istiocontrolplane.proto

```diff
@@ -135,6 +135,24 @@ message SidecarInjectorConfiguration {
     BaseKubernetesResourceConfig deployment = 1;
     // Service spec
     Service service = 2;
+    // SidecarInjectionTemplates spec
```

Comment:
Since this is human readable comment, I'd suggest like this:
```suggestion
    // Sidecar injection templates spec
```

Also, we could probably write a few sentences that with this field the default sidecar and/or gateway injection template can be overriden and also custom templates can be defined, which then can be used on pods with the inject.istio.io/templates=custom annotations.

Reply:
I think the comment is not yet updated as suggested.